docs/source/guides/staking_guide.rst

```diff
@@ -215,8 +226,8 @@ attached a Worker node to your Staker, so you still have to do it!
 Bond an Ursula to a Staker
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-After you create an Ethereum node for your worker (see below about the worker),
-you can set the worker. Stakes which don't yet have a worker will be
+After you create an Ethereum node for your staker (see below about the worker),
```

Comment:
I think it's actually correct to say the worker has the Ethereum node, right?

Reply:
YEs, but this step is referred to the Staker, since she is setting now her worker.

Reply:
It's subtle - but the original intention is to indicate that the staker must know about the worker's address to call `set-worker`.